src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -216,6 +216,14 @@
       <Versions>9</Versions>
     </RuntimeGroup>
 
+    <RuntimeGroup Include="rocky">
+      <Parent>rhel</Parent>
+      <Architectures>x64;arm64</Architectures>
+      <Versions>8;8.4;9</Versions>
```

Comment:
Is there significance to 8.4? Seems like the build is compelling the parent is missing

Reply:
My experience with RHEL is that we really want to avoid adding minors here. There's going to be a Rocky/RHEL 8.x release every 6 months or so. (It's tiring enough on Fedora where there's a major bump every 6 months.) It was entirely unnecessary for RHEL 8.4, and I think we should do the same for Rocky.

Reply:
Thanks for pointing this out.  I wasn't entirely sure of the significance of including the minor version, either.  I merged the original Rocky Linux patches and one had `8` and the other `8.4`, IIRC.  If you see no reason to have the minor version listed, I can remove it from the PR.

Reply:
@wfurt / @omajid FYI, I just removed the 8.4 from the PR